modules/services/pbgopy.nix

```
@@ -5,18 +5,41 @@ with lib;
 let
   cfg = config.services.pbgopy;
   package = pkgs.pbgopy;
+
+  commandLine = concatStringsSep " " ([ "${package}/bin/pbgopy serve" ]
+    ++ optional (cfg.port != 9090) "--port ${toString cfg.port}"
+    ++ optional (cfg.cache.ttl != "24h") "--ttl ${cfg.cache.ttl}"
+    ++ optional (cfg.httpAuth != "") "--basic-auth ${cfg.httpAuth}");
```

```suggestion
    ++ optional (cfg.httpAuth != "") "--basic-auth ${escapeShellArg cfg.httpAuth}");
```

How about escaping the username:password combination?
The documentation would also need to mention the username and password are escaped.

---

Done :+1: